### ISSUE TITLE
最新匿名短信系统网站源码/匿名信卡密版本源码

### ISSUE THREAD

**Issue by wukongymw** [wukongymw/wukongymw#114]
<div id="announcement"><div>
 <h2>重要公告</h2>
    <p>
        加入我们的 Telegram 群组以获取最新消息和支持！ 
        <br>
        📢 <a href="https://t.me/wukongym" target="_blank">点击这里加入 TG 群</a>
        <br><br>
        如需帮助，请联系唯一客服：
        <br>
        🤖 <a href="https://t.me/wukongymw" target="_blank">点击这里联系客服</a>
    </p>
  </div>
</div>

匿名短信发送源码，成本0.1一条65字源码限制了120字最大，卡密版本，可自己后台生成卡密 可自己对接代刷或者社区卖卡！表白祝福啥的

1.系统环境

-php要求 5.6 过高部分函数会失效无法使用

-mysql 无要求，数据库手动导入

2.配置问题

-导入数据库后，在config.php文件夹里修改数据库信息

3.api对接问题

-搭建好之后，在ajax.php文件里

-修改第 6 行的API key,为您拿到的key

key获取地址：https://jhapi.rjk66.cn/user/key

短信成本0.12需要冲点数喔

4.后台入口可自定义

-修改 admin 目录名即可

<a href="https://cdn.wukongymw.com/wp-content/uploads/2023/03/1677916938-ea4c74ae6815ef1.png"><img src="https://cdn.wukongymw.com/wp-content/uploads/2023/03/1677916938-ea4c74ae6815ef1.png" width="1008" height="713" class="aligncenter size-full wp-image-198438" style="border-radius:12px; padding:5px; display:block; clear:both; margin:0 auto;"></a>
